Unable to run vue3-sample-app, getting multiple error

### Core Library

MSAL.js v2 (@azure/msal-browser)

### Core Library Version

2.26.0

### Wrapper Library

Not Applicable

### Wrapper Library Version

N/A

### Description

After running ```npm install``` without issues
running ```npm start``` and getting multiple error then trying to access the site:

```log
  vite v2.9.12 dev server running at:

  > Local: http://localhost:3000/
  > Network: use `--host` to expose

  ready in 553ms.

✘ [ERROR] Could not resolve "element-plus/es/components"

    node_modules/element-plus/es/index.js:3:14:
      3 │ export * from 'element-plus/es/components';
        ╵               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  Importing the directory "./es/components" is not supported:

    node_modules/element-plus/package.json:22:11:
      22 │     "./*": "./*"
         ╵            ~~~~~

  You can mark the path "element-plus/es/components" as external to exclude it from the bundle,
  which will remove this error.
````

```npm list``` output:
```
+-- @azure/msal-browser@2.26.0
+-- @vitejs/plugin-vue@1.10.2
+-- element-plus@1.1.0-beta.23
+-- typescript@4.7.4
+-- vite@2.9.12
+-- vue-router@4.0.16
+-- vue-tsc@0.3.0
`-- vue@3.2.37
```

### Error Message


```
  vite v2.9.12 dev server running at:

  > Local: http://localhost:3000/
  > Network: use `--host` to expose

  ready in 553ms.

✘ [ERROR] Could not resolve "element-plus/es/components"

    node_modules/element-plus/es/index.js:3:14:
      3 │ export * from 'element-plus/es/components';
        ╵               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  Importing the directory "./es/components" is not supported:

    node_modules/element-plus/package.json:22:11:
      22 │     "./*": "./*"
         ╵            ~~~~~

  You can mark the path "element-plus/es/components" as external to exclude it from the bundle,
  which will remove this error.

9:54:23 [vite] error while updating dependencies:
Error: Build failed with 76 errors:
node_modules/element-plus/es/component.js:1:24: ERROR: Could not resolve "element-plus/es/components/affix"
node_modules/element-plus/es/component.js:2:24: ERROR: Could not resolve "element-plus/es/components/alert"
node_modules/element-plus/es/component.js:3:31: ERROR: Could not resolve "element-plus/es/components/autocomplete"
node_modules/element-plus/es/component.js:4:25: ERROR: Could not resolve "element-plus/es/components/avatar"
node_modules/element-plus/es/component.js:5:26: ERROR: Could not resolve "element-plus/es/components/backtop"
...
    at failureErrorWithLog (C:\ws\Harness\msal.js\samples\msal-browser-samples\vue3-sample-app\node_modules\esbuild\lib\main.js:1605:15)
    at C:\ws\Harness\msal.js\samples\msal-browser-samples\vue3-sample-app\node_modules\esbuild\lib\main.js:1251:28
    at runOnEndCallbacks (C:\ws\Harness\msal.js\samples\msal-browser-samples\vue3-sample-app\node_modules\esbuild\lib\main.js:1034:63)
   7 at buildResponseToResult (C:\ws\Harness\msal.js\samples\msal-browser-samples\vue3-sample-app\node_modules\esbuild\lib\main.js:1249:7)
    at C:\ws\Harness\msal.js\samples\msal-browser-samples\vue3-sample-app\node_modules\esbuild\lib\main.js:1358:14
    at C:\ws\Harness\msal.js\samples\msal-browser-samples\vue3-sample-app\node_modules\esbuild\lib\main.js:666:9
    at handleIncomingPacket (C:\ws\Harness\msal.js\samples\msal-browser-samples\vue3-sample-app\node_modules\esbuild\lib\main.js:763:9)
    at Socket.readFromStdout (C:\ws\Harness\msal.js\samples\msal-browser-samples\vue3-sample-app\node_modules\esbuild\lib\main.js:632:7)
    at Socket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
Vite Error, /node_modules/.vite/deps/element-plus.js?v=673559b5 optimized info should be defined
```



### Msal Logs

_No response_

### MSAL Configuration

```javascript
N/A
```


### Relevant Code Snippets

```javascript
N/A
```


### Reproduction Steps

1. download sample
2. run  ```npm install @azure/msal-browser```
3. run ```npm install```
4. run ```npm start```
5. try to access the localhost address
6. see output error

### Expected Behavior

run without errors

### Identity Provider

Azure AD / MSA

### Browsers Affected (Select all that apply)

Edge

### Regression

_No response_

### Source

External (Customer)

--- a/samples/msal-browser-samples/vue3-sample-app/package.json
+++ b/samples/msal-browser-samples/vue3-sample-app/package.json
@@ -7,10 +7,10 @@
     "serve": "vite preview"
   },
   "dependencies": {
-    "@azure/msal-browser": "^2.18.0",
-    "element-plus": "^1.1.0-beta.20",
-    "vue": "^3.2.20",
-    "vue-router": "^4.0.11"
+    "@azure/msal-browser": "^2.26.0",
+    "element-plus": "^2.2.6",
+    "vue": "^3.2.37",
+    "vue-router": "^4.0.16"
   },
   "devDependencies": {
     "@vitejs/plugin-vue": "^1.9.3",

